src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -193,9 +193,20 @@ class BankAccountStep extends React.Component {
                             <MenuItem
                                 icon={Paycheck}
                                 title={this.props.translate('bankAccount.connectManually')}
+                                disabled={!this.props.user.validated}
                                 onPress={() => setBankAccountSubStep(CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL)}
                                 shouldShowRightIcon
                             />
+                            {!this.props.user.validated && (
```

Comment:
👋 I'm not sure if this was true at the time of creation of this PR, but this might have caused an issue in https://github.com/Expensify/App/issues/18156
This is an edge case, which can be reproduced only if you clear cookies and site data and then sign in again

Before the API call to get user info is completed,`this.props.user.validated` is `undefined`, which causes `validateAccountError` to display for a split second while API call is in progress
We resolved this by displaying a loading indicator while the API call is in progress, but that didn't exist when this PR was created, so `{this.props.user.validated === false && (` would have avoided this